client/container_copy.go

```diff
@@ -30,6 +30,7 @@ func (cli *Client) ContainerStatPath(ctx context.Context, containerID, path stri
 }
 
 // CopyToContainer copies content into the container filesystem.
+// Note that `content` must be a Reader for a TAR
```

Comment:
The `Reader` from this sentence refers to `io.Reader`

I was going to change "TAR file" to "TAR archive" but 1.) That's like saying "ATM machine" and it makes people upset, and 2.) @AkihiroSuda suggested just "TAR" which sounds a little strange but makes sense when you expand the acronym: "A Reader for a Tape Archive"